Cargo.toml

```diff
@@ -25,14 +25,18 @@ async-stream = "0.3.5"
 async-trait = "0.1.77"
 derive_builder = "0.13.0"
 futures = "0.3.30"
+reqwest = { version = "0.12.7", features = ["json"] }
 serde = { version = "1.0.195", features = ["derive"] }
 serde_json = "1.0.111"
 thiserror = "1.0.56"
 tokio = { version = "1.35.1", features = ["sync", "time", "rt", "macros"] }
 tokio-stream = { version = "0.1.14" }
 tracing = "0.1.40"
 
-near-indexer-primitives = "0.23.0"
+# Bug with deserialization of the transactions it should be fixed in the next release  0.27.0
+# near-indexer-primitives = "0.26.0"
+# for now we use the forked version
+near-indexer-primitives = { git = 'https://github.com/kobayurii/nearcore.git', branch = "2.2.1-fork1" }
```

Comment:
Reminder: this is the blocker from merging this change into the 0.7.x branch and releasing a new version

Reply:
solved